### PR TITLE
floder structure

### PR DESCRIPTION
**Pull request by ReshmaScriptlanes1999**

### CODE REVIEW

File: src/app/component/scriptlanesheading/scriptlanesheading.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-scriptlanesheading',
+  templateUrl: './scriptlanesheading.component.html',
+  styleUrls: ['./scriptlanesheading.component.css']
+})
+export class ScriptlanesheadingComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (ayushmishra1803) [inline src/app/component/scriptlanesheading/scriptlanesheading.component.ts]: please remove this method if not in used anywhere

File: src/app/page/onlinetest/component/options/options.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-options',
+  templateUrl: './options.component.html',
+  styleUrls: ['./options.component.css']
+})
+export class OptionsComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (ayushmishra1803) [inline src/app/page/onlinetest/component/options/options.component.ts]: please remove all the unused methods

File: src/app/page/onlinetest/component/questionheading/questionheading.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-questionheading',
+  templateUrl: './questionheading.component.html',
+  styleUrls: ['./questionheading.component.css']
+})
+export class QuestionheadingComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (ayushmishra1803) [inline src/app/page/onlinetest/component/questionheading/questionheading.component.ts]: please remove all the unused methods

File: src/app/page/onlinetest/component/questions/questions.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-questions',
+  templateUrl: './questions.component.html',
+  styleUrls: ['./questions.component.css']
+})
+export class QuestionsComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (ayushmishra1803) [inline src/app/page/onlinetest/component/questions/questions.component.ts]: please remove all the unused methods

File: src/app/page/onlinetest/onlinetest.component.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-onlinetest',
+  templateUrl: './onlinetest.component.html',
+  styleUrls: ['./onlinetest.component.css']
+})
+export class OnlinetestComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (ayushmishra1803) [inline src/app/page/onlinetest/onlinetest.component.ts]: please remove all the unused methods

File: src/app/page/onlinetest/onlinetest.module.ts [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { OnlinetestRoutingModule } from './onlinetest-routing.module';
+
+
+@NgModule({
+  declarations: [],
```
Comment (ayushmishra1803) [inline src/app/page/onlinetest/onlinetest.module.ts]: please declare online test component in declarations array

Comment (ayushscriptlanes) [inline src/app/page/onlinetest/onlinetest.module.ts]: please include component in the declarations

File: src/app/page/onlinetest/onlinetest.component.ts (hunk 5 above)
Comment (ayushscriptlanes) [inline src/app/page/onlinetest/onlinetest.component.ts]: please remove all the unused method

File: src/app/page/onlinetest/onlinetest-routing.module.ts [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+const routes: Routes = [];
```
Comment (ayushscriptlanes) [inline src/app/page/onlinetest/onlinetest-routing.module.ts]: add respective routes

File: src/app/page/onlinetest/component/questions/questions.component.ts (hunk 4 above)
Comment (ayushscriptlanes) [inline src/app/page/onlinetest/component/questions/questions.component.ts]: please remove all the unused method

File: src/app/page/onlinetest/component/questionheading/questionheading.component.ts (hunk 3 above)
Comment (ayushscriptlanes) [inline src/app/page/onlinetest/component/questionheading/questionheading.component.ts]: please remove all the unused method

File: src/app/page/onlinetest/component/options/options.component.ts (hunk 2 above)
Comment (ayushscriptlanes) [inline src/app/page/onlinetest/component/options/options.component.ts]: please remove all the unused method

File: src/app/component/scriptlanesheading/scriptlanesheading.component.ts (hunk 1 above)
Comment (ayushscriptlanes) [inline src/app/component/scriptlanesheading/scriptlanesheading.component.ts]: please remove all the unused method

**Review by ayushscriptlanes — CHANGES_REQUESTED** [review]

File: src/app/page/onlinetest/onlinetest-routing.module.ts [hunk 8]
```diff
@@ -1,7 +1,21 @@
-import { NgModule } from '@angular/core';
+import { Component, NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { OnlinetestComponent } from './onlinetest.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
```
Comment (ayushmishra1803) [inline src/app/page/onlinetest/onlinetest-routing.module.ts:6]: we are going to create dynamic routes

**Review by ayushmishra1803 — CHANGES_REQUESTED** [review]

**Review by ayushscriptlanes — APPROVED** [review]

**Pull request merged**